packages/stark-build/config/webpack-partial.common.js

```diff
@@ -80,7 +89,9 @@ module.exports = metadata => {
 						{
 							loader: "tslint-loader",
 							options: {
-								typeCheck: false // FIXME remove this line once the type checking issues are gone (cfr FIXME above)
+								typeCheck: false, // FIXME remove this line once the type checking issues are gone (cfr FIXME above)
+								tsconfig: buildUtils.ANGULAR_APP_CONFIG.buildOptions.tsconfig || "tsconfig.json",
+								configFile: tslintLoaderFilePath
```

Comment:
Can you add a comment to explain what we are doing here? And could you add the links related to the issue?